tree/dataframe/inc/ROOT/RDFActionHelpers.hxx

```diff
@@ -363,21 +373,20 @@ public:
    using Result_t = ::TGraph;
 
 private:
-   std::shared_ptr<::TGraph> fResultGraph;
-   std::unique_ptr<ROOT::TThreadedObject<::TGraph>> fTo;
+   std::vector<::TGraph*> fGraphs;
 
 public:
    FillTGraphHelper(FillTGraphHelper &&) = default;
    FillTGraphHelper(const FillTGraphHelper &) = delete;
 
    // The last parameter is always false, as at the moment there is no way to propagate the parameter from the user to this method
    FillTGraphHelper(const std::shared_ptr<::TGraph> &g, const unsigned int nSlots)
-      : fResultGraph(g), fTo(new ROOT::TThreadedObject<::TGraph>(*g))
+      : fGraphs(nSlots, nullptr)
    {
-      fTo->SetAtSlot(0, g);
+      fGraphs[0] = g.get();
```

Comment:
same comment as per the `shared_ptr<TH1D>` above: if `FillTGraphHelper` does not retain ownership, it should not take it by `shared_ptr` in its ctor, it's misleading